build-system/tasks/e2e/index.js

```diff
@@ -25,6 +25,7 @@ const Mocha = require('mocha');
 const tryConnect = require('try-net-connect');
 const {cyan} = require('ansi-colors');
 const {execOrDie, execScriptAsync} = require('../../exec');
+const {reportTestStarted} = require('../runtime-test/status-report');
```

Comment:
nit: move `status-report.js` file out of `tasks/runtime-test`

Reply:
Done